modules/transactions.js

```diff
@@ -1,23 +1,23 @@
 'use strict';
 
-var async = require('async');
-var ByteBuffer = require('bytebuffer');
-var constants = require('../helpers/constants.js');
-var crypto = require('crypto');
-var extend = require('extend');
-var genesisblock = null;
-var OrderBy = require('../helpers/orderBy.js');
-var Router = require('../helpers/router.js');
-var sandboxHelper = require('../helpers/sandbox.js');
-var schema = require('../schema/transactions.js');
-var slots = require('../helpers/slots.js');
-var sql = require('../sql/transactions.js');
-var TransactionPool = require('../logic/transactionPool.js');
-var transactionTypes = require('../helpers/transactionTypes.js');
-var Transfer = require('../logic/transfer.js');
+var _ = require('lodash'),
```

Comment:
Please don't do parallel assignment here. See: https://github.com/airbnb/javascript/tree/es5-deprecated/es5#variables

Yes we are moving to new standard, but for now es5 is it.